src/data/equiv/local_equiv.lean

```diff
@@ -520,6 +603,59 @@ by ext x; simp [ext_iff]; tauto
 
 end prod
 
+/-- Combine two `local_equiv`s using `set.piecewise`. The definition assumes
+`e.is_image s t` and `e'.is_image s t`. -/
```

Comment:
Please recall in the docstring what it does, i.e., what the source is and how it behaves on each pieces of the source.